### ISSUE TITLE
Clarify DPE Usage of PCRs

### ISSUE THREAD

**Issue by jhand2** [chipsalliance/caliptra-rtl#1]
tl;dr - Proposal is to not change the current PCR count, and keep it at 32

Currently, the hardware spec says that Caliptra will have 32 hardware PCRs. This breaks down to

* 24 PCRs for DPE. One for each "Journey" TCI.
* 8 for all other Caliptra usages

There has been a proposal to add an additional 24 PCRs for storing the "Current" value for each TCI. This is the most recent value that has been stored in a DPE TCI. These PCRs would be clearable by Runtime Firmware and never locked.

I would like to instead propose that we store the DPE "Current" TCIs in DCCM SRAM for the following reasons:

1. Additional PCRs would add ~10K flops. The corresponding area increase could impact SoC integrators' ability to fit Caliptra.
2. If "Current" values are stored in PCRs, they need to be clearable. This means that Runtime Firmware has the same ability to clear "Current" values regardless of whether they're stored in PCRs or DCCM.
3. A corrupt or malicious Current TCI value can be detected by interpreting the corresponding journey.

**Comment by bharatpillilli** [comment]
Implemented, validated & released - closing
